packages/app-desktop/main-html.js

```diff
@@ -157,6 +157,8 @@ app().start(bridge().processArgv()).then((result) => {
 		}
 	}
 
-	// In dev, we leave the app open as debug statements in the console can be useful
-	if (env !== 'dev') bridge().electronApp().exit(1);
+	// In dev, we give the option to leave the app open as debug statements in the
+	// console can be useful
+	const canIgnore = env === 'dev';
+	bridge().electronApp().handleAppFailure(error, canIgnore);
```

Comment:
The code several lines above this also handles showing an error message to the user.

As `handleAppFailure` also does this, the original code should be instead used to create a better error message in the `handleAppFailure` dialog.

Reply:
Changed.